README.md

```diff
@@ -10,17 +10,19 @@ and [UDSinks](https://numaproj.github.io/numaflow/sinks/user-defined-sinks/) in
 from pynumaflow.function import Messages, Message, Datum, UserDefinedFunctionServicer
 
 
-def map_handler(key: str, datum: Datum) -> Messages:
+def cat_handler(key: str, datum: Datum) -> Messages:
```

Comment:
let's not call it cat? name it function_handler?

Reply:
Sure